tgui/packages/tgui/components/NanoMap.js

```diff
@@ -134,8 +134,8 @@ const NanoMapMarker = (props, context) => {
     tooltip,
     color,
   } = props;
-  const rx = x * 2 * zoom - 3;
-  const ry = y * 2 * zoom - 3;
+  const rx = x * 2 * zoom - zoom - 3;
+  const ry = y * 2 * zoom - zoom - 3;
```

Comment:
Is the double-takeaway zoom correct here, and if so, can you stick some brackets around the maths in places to make it easier to read?

Reply:
Yes and okay.